generate_report.py

```diff
@@ -303,55 +321,105 @@ def OutputReportFile(mode: str, spec: typing.Dict[str, float], vs: float,
       ];
       if(ele != null){
         ele.style.display = "block";
-        sda = document.getElementById('sda_show');
-        if(sda!=null){sda.style.display = "none";}
-         scl = document.getElementById('scl_show');
-        if(scl!=null){scl.style.display = "none";}
+        rect = document.getElementById(x + "_rect");
+        if(rect!=null){rect.style.display = "inline";}
+        line = document.getElementById(x + "_line");
+        if(line!=null){line.style.display = "inline";}
+        poly = document.getElementById(x + "_poly");
+        if(poly!=null){poly.style.display = "inline";}
         fields.forEach(function(item, index, array) {
           if (item != x){
             ele1 = document.getElementById(item);
             if(ele1!=null){ele1.style.background = "white";}
             ele2 = document.getElementById(item + "_hide");
-            if(ele2!=null){ele2.style.display = "none";}
+            if(ele2!=null){
+              ele2.style.display = "none";
+              rect2 = document.getElementById(item + "_rect");
```

Comment:
use something like
```
.invisible {
  display: none;
}
```
and `ele2.toggleClass('invisible')` or `ele2.addClass('invisible')` in js code